docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -79,10 +79,111 @@ on our `feedback form <https://docs.google.com/forms/d/e/1FAIpQLSeCED9TFQsH-1u4Q
+identities and capturing this is important. However, it creates a statistical 
+issue without an easy answer - how do you handle overlapping groups? 
+
+To understand the options below, let's provide an example of a single 
+person who selected both "Black or African American" and "Asian" for 
+their race. 
+
+The first option is to include all 
+relevant race/ethnicity groups. So for this example, you would have 
+groups for "Black or African American", "Asian", and "Both Black or African 
+American and Asian". If your dataset is large 
+enough to support having this level of granularity in groups, this 
+approach can work well. However, often this leads to issues with 
+small sample sizes. 
+
+The second option is to exclude everyone who selected multiple race 
+or ethnicity groups. So we would just not count our example individual. We do not 
+recommend this approach. 
+
+The third is to create a large, "multiracial" group. The resulting 
+groups would then be "Black or African American", "Asian", and "Multiracial". We also do 
+not recommend this approach as the resulting group is generally 
+too diverse to have any meaningful conclusions about.
+
+The fourth is to count people in all race/ethnicity groups they 
+selected. Therefore in our example, the individual would be 
+counted twice - once in "Black or African American" and once in "Asian". This 
+can lead to double counting in the data which might be more or 
+less important depending on the size of the multiracial group 
+in the dataset. This might be a reasonable option.
+
+The fifth option is to run analyses with 
+both a more limited race/ethnicity group, and then with a larger, 
+multiracial group. For example, you would have categories of "Asian" 
+and "Asian alone" both existing. "Asian" would include anyone who 
+selected Asian, including the multiracial person in our example, 
+and "Asian alone" would be people who only identify as 
+Asian, excluding the example individual. Often people will present 
+anlyses for both of these groups. This is also a reasonable 
+option. 
+
+Last, you can attempt to crosswalk individuals into a single 
+race/ethnicity group. There have been multiple attempts to do 
+this based on studies that allow respondents to select all racial 
+categories they identify with and then to pick a single one they 
+most identify with. [Liebler_2008]_ Therefore, the example individual 
+would be placed in a single racial group - either "Black or African 
+American" or "Asian" based on their other data. If this is 
+feasible based on the data present, it is also a reasonable approach. 
+
+Often, someone will have made the decision about how to handle 
+race/ethnicity categories and multiracial individuals before you 
+receive the data. In this case, try to find out what was assumed 
+and note it appropriately in your limitations as needed. There is 
+more information on this in the section below. 
```

Comment:
I feel like this last paragraph could be made more prominent, since this is almost always the situation we are in. Maybe we should say this first, then say "there are a number of different ways your data might code race and ethnicity" and list the options you have above?